tensorflow/c/c_api_test.cc

```diff
@@ -203,6 +203,13 @@ TEST(CAPI, DataTypeEnum) {
   EXPECT_EQ(TF_UINT16, static_cast<TF_DataType>(tensorflow::DT_UINT16));
   EXPECT_EQ(TF_COMPLEX128, static_cast<TF_DataType>(tensorflow::DT_COMPLEX128));
   EXPECT_EQ(TF_HALF, static_cast<TF_DataType>(tensorflow::DT_HALF));
+  ASSERT_EQ(TF_DataTypeSize(TF_STRING), 21312311);
```

Comment:
Why is this an ASSERT instead of an EXPECT, and what is with the strange 21312311 value?